.github/workflows/release-build.yaml

```diff
@@ -0,0 +1,84 @@
+name: Docker Build Publish
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
```

Comment:
If we remove this, would it prohibit/remove the creation of `vX.X.X` tags for the time being. We will not be able to click this button until the Jenkins prod deployment is deprectated b/c it will cause those to fail if the tag already exists... thoughts @emuvente ???